lighthouse-cli/test/smokehouse/test-definitions/forms/form-expectations.js

```diff
@@ -0,0 +1,307 @@
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/**
+ * @type {Array<Smokehouse.ExpectedRunnerResult>}
+ * Expected Lighthouse artifacts from Form gatherer
+ */
+const expectations = [
+  {
+    artifacts: {
+      Forms: [
+        {
+          attributes: {
+            id: 'checkout',
+            name: 'checkout',
+            autocomplete: 'on',
+            devtoolsNodePath: '0,HTML,1,BODY,0,FORM',
+          },
+          inputs: [
+            {
+              id: '',
+              nodeName: 'TEXTAREA',
+              name: 'name_cc',
+              placeholder: '',
+              autocomplete: 'cc-name',
+              devtoolsNodePath: '0,HTML,1,BODY,0,FORM,1,TEXTAREA',
+            },
+            {
+              id: '',
+              nodeName: 'INPUT',
+              name: 'CCNo',
+              placeholder: '',
+              autocomplete: 'cc-number',
+              devtoolsNodePath: '0,HTML,1,BODY,0,FORM,4,INPUT',
+            },
+            {
+              id: '',
+              nodeName: 'SELECT',
+              name: 'CCExpiresMonth',
+              autocomplete: 'cc-exp-month',
+              devtoolsNodePath: '0,HTML,1,BODY,0,FORM,7,SELECT',
+            },
+            {
+              id: '',
+              nodeName: 'SELECT',
+              name: 'CCExpiresYear',
+              autocomplete: 'cc-exp-year',
+              devtoolsNodePath: '0,HTML,1,BODY,0,FORM,9,SELECT',
+            },
+            {
+              id: '',
+              nodeName: 'INPUT',
+              name: 'cvc',
+              placeholder: '',
+              autocomplete: 'cc-csc',
+              devtoolsNodePath: '0,HTML,1,BODY,0,FORM,12,INPUT',
+            },
+          ],
+          labels: [
+            {
+              id: 'cardName',
+              nodeName: 'LABEL',
+              for: 'name_cc',
+              devtoolsNodePath: '0,HTML,1,BODY,0,FORM,0,LABEL',
+            },
+            {
+              id: 'cardNumber',
+              nodeName: 'LABEL',
+              for: 'CCNo',
+              devtoolsNodePath: '0,HTML,1,BODY,0,FORM,3,LABEL',
+            },
+            {
+              id: 'expDate',
+              nodeName: 'LABEL',
+              for: 'CCExpiresMonth',
+              devtoolsNodePath: '0,HTML,1,BODY,0,FORM,6,LABEL',
+            },
+            {
+              id: 'cvc',
+              nodeName: 'LABEL',
+              for: 'cvc',
+              devtoolsNodePath: '0,HTML,1,BODY,0,FORM,11,LABEL',
```

Comment:
these devtools node paths are going to be extremely brittle if we ever change the smoke test to add new cases.

is the primary motivation here just to make sure we capture *something* for it? if so, maybe we use `/FORM,\d+,LABEL$/` or similar?

Reply:
@patrickhulce can I use snippet then?

Reply:
The id alone is enough to identify the node, and identification is really the only reason to use `devtoolsNodePath` or `snippet` in the smoke tests. You can just drop all `devtoolsNodePath` here.

Reply:
@connorjclark mainly using snippet for the screenshot of the element in the audit